parsel/selector.py

```diff
@@ -27,6 +28,10 @@ class CannotRemoveElementWithoutParent(Exception):
     pass
 
 
+class CannotDropElementWithoutParent(Exception):
```

Comment:
Would it make sense to inherit it from the previous exception, so that exception handling is backwards compatible? Or would it be too convoluted?

Reply:
I think it make sense, done :)